dask/dataframe/partitionquantiles.py

```diff
@@ -238,7 +238,7 @@ def create_merge_tree(func, keys, token):
 def percentiles_to_weights(qs, vals, length):
     """Weigh percentile values by length and the difference between percentiles
 
-    >>> percentiles = np.array([0, 25, 50, 90, 100])
+    >>> percentiles = np.array([0., 25., 50., 90., 100.])
```

Comment:
Not sure if this method needs to raise a clearer error if dtype=int

Reply:
seems like the failure on master might indicate a change in numpy since these pieces of code are several years old.